docs/transpile/defaults_and_configuration_options.mdx

```diff
@@ -0,0 +1,150 @@
+
+- `optimization_level=0`: Trivial optimization, which maps the circuit to the backend with no explicit optimization (except whatever optimizations the mapper does).
+- `optimization_level=1`: Light optimization, which maps the circuit and does simple optimizations by collapsing adjacent gates. This is the default value.
+- `optimization_level=2`: Medium optimization, including a noise-adaptive layout and a gate-cancellation procedure based on gate commutation relationships.
+- `optimization_level=3`: Heavy optimization, which performs the optimization included in levels 1 - 2 and resynthesizes two-qubit gate blocks in the circuit.
+
+<Table>
+  <thead>
+    <Tr>
+      <Th>Optimization Level</Th>
+      <Th>Description</Th>
+    </Tr>
+  </thead>
+  <tbody>
+    <Tr>
+      <Td>0</Td>
+      <Td>
+        No optimization: typically used for hardware characterization
+        - Basic translation
+        - Layout (as specified)
+        - Routing (stochastic swaps)
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>1</Td>
+      <Td>
+        Light optimization (default):
+
+        -   Layout (trivial → vf2 → SabreLayout if routing is required)
+        -   Routing (SabreSWAPs if needed)
+        -   1Q gate optimization
+        -   Error suppression: dynamical decoupling
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>2</Td>
+      <Td>
+        Medium optimization:
+          - Layout/Routing: Optimization level 1 (without trivial) + heuristic optimized with greater
+        search depth and trials of optimization function
+        -   Commutative cancelation
+        -   Error suppression: dynamical decoupling
+      </Td>
+    </Tr>
+    <Tr>
+      <Td>3 (default)</Td>
+      <Td>
+        High Optimization:
+
+        - Optimization level 2 + heuristic optimized on layout/routing further with greater effort/trials
+        - Two qubit KAK optimization
+        - Error suppression: dynamical decoupling
+      </Td>
+    </Tr>
+  </tbody>
+</Table>
+
+Example, specifying `optimization_level=0`: 
+
+```python
+    from qiskit import QuantumCircuit, transpile
+    from qiskit.visualization import plot_circuit_layout
+    from qiskit.providers.fake_provider import FakeVigo
+    backend = FakeVigo()
+
+    ghz = QuantumCircuit(3, 3)
+    ghz.h(0)
+    ghz.cx(0,range(1,3))
+    ghz.barrier()
+    ghz.measure(range(3), range(3))
+
+    new_circ_lv0 = transpile(ghz, backend=backend, optimization_level=0)
+    plot_circuit_layout(new_circ_lv0, backend)
+```
+
+
+### Approximation degree
+
+** ~~~Question about this: "approximation_degree (float) - Heuristic dial used for circuit approximation (1.0=no approximation, 0.0=maximal approximation). The default value is 1.0."
+
+It looks like the value is a float in the range [0 - 1]. Should we clarify that in the description?  What do the "in between" measurements mean?**
+
+### Other importatnt options?
```

Comment:
I think `seed_transpiler`,  `initial_layout` and the `*_method` arguments are the next most commonly used arguments.